R/change.R

```diff
@@ -106,7 +106,7 @@ arc <- function(years, values, denominator = 100) {
 
   # calculate ARC
   output <- denominator *
-            log(values - values_offset) /
+            log(values / values_offset) /
```

Comment:
Maybe add a citation to the details? Another possible one is pg 12 of the Preston Demography book.

Reply:
I'm guessing it was just a typo previously and should have been `log(values) - log(values_offset)`